tests/Integrations/PCNTL/PCNTLTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\PCNTL;
+
+use DDTrace\Tests\Common\IntegrationTestCase;
+use DDTrace\Tests\Common\SpanAssertion;
+
+const ACCEPTABLE_TEST_EXECTION_TIME_S = 2;
```

Comment:
Is this in seconds?

Reply:
I changed to hundreds of milliseconds https://github.com/DataDog/dd-trace-php/pull/1196/commits/768c1b90d6708686d2901ffdea270ad4d7fda785. Good catch that was a left over from another test that then was removed.

Reply:
Sorry, to clarify I was curious if the `_S` suffix referred to seconds. But I think the units are off now since `microtime(true)` will return seconds as a float with microsecond accuracy. But it looks like the comparisons are comparing second resolution `$end - $start` instead of millisecond resolution, `($end - $start) * 1000`, right?